From: Jiri Slaby <jslaby@suse.cz>
Date: Tue, 26 Jan 2016 14:13:01 +0100
Subject: kgr: log when modifying kernel
Patch-mainline: not yet, kgraft
References: fate#317827

Drop a message to the log that a patch is being applied/reverted.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 kernel/kgraft.c |    3 +++
 1 file changed, 3 insertions(+)

--- a/kernel/kgraft.c
+++ b/kernel/kgraft.c
@@ -942,6 +942,9 @@ int kgr_modify_kernel(struct kgr_patch *
 		goto err_unlock;
 	}
 
+	pr_info("kgr: %sing patch '%s'\n", revert ? "revert" : "apply",
+			patch->name);
+
 	set_bit(0, kgr_immutable);
 	wmb(); /* set_bit before kgr_mark_processes */
 
